source-code/core/app/src/wrappers/withSolidReactivity.test.ts

```diff
@@ -0,0 +1,325 @@
+import { describe, it, expect } from "vitest"
+import { createInlang } from "../createInlang.js"
+import { createMockNodeishFs } from "@inlang/plugin/test"
```

Comment:
@samuelstroschein should this import not come from `@inlang/module/test` or `@inlang/app/test`?

Reply:
Ok, I'll re-export it and it should be fine. I just was confused that I need to import this form `plugin` inside the `sdk` app.